pkg/reconciler/stats_reporter.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"context"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"knative.dev/pkg/metrics"
+)
+
+const (
+	knativeServingChangeCountName = "knativeserving_change_count"
+)
+
+var (
+	knativeServingChangeCountStat = stats.Int64(
+		knativeServingChangeCountName,
+		"Number of KnativeServing changes including creation, edit and deletion",
```

Comment:
[nit] "Number of changes to the KnativeServing Custom Resource where a change can be create, update, or delete."